notebooks/community/ml_ops/stage6/get_started_with_xgboost_model_online.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>Can we add 1-2 sentences on what the prediction means? </p><p>This returns <code>predictions=[0.0004718601703643799]</code>which I didn't really understand what it is. Isn't predicting the type of Iris among 3 labels?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
I ran the new version and got `Prediction(predictions=[0.0], deployed_model_id='5334711670713352192', model_version_id='1')` as a result. Is this expected?